modern/src/settings/CalendarPage.js

```diff
@@ -66,9 +66,11 @@ const CalendarPage = () => {
                 label={t('sharedName')}
               />
               <DropzoneArea
+                dropzoneText={t('sharedDropzoneText')}
                 filesLimit={1}
                 onChange={handleFiles}
                 alertSnackbarProps={components.MuiSnackbar.defaultProps}
```

Comment:
We probably don't need this if alerts are disabled.

Reply:
you are right! we really don't need it. done!